libs/weblit-static.lua

```diff
@@ -51,9 +52,10 @@ return function (rootPath)
       return bundle.readfile(pathJoin(prefix, path))
     end
   else
+    pathUtils = require('path')
```

Comment:
This assignment seems to go to a global variable as do all subsequent requests to it because the local goes out of scope at the preceding else. I would recommend making path utils local to the top level of this file and then removing the then-shadowing local declaration in the luvi bundle section and making it a simple assignment instead.